lib/spree/localized_number.rb

```diff
@@ -30,13 +33,14 @@ def localize_number(*attributes)
             old_setter.bind(self).call(number)
           end
         end
+      end
+    end
 
-        define_method(:validate_localizable_number) do
-          return unless Spree::Config.enable_localized_number?
-
-          @invalid_localized_number.andand.each do |error_attribute|
-            errors.set(error_attribute, [I18n.t('spree.localized_number.invalid_format')])
-          end
+    def define_validate_method
+      define_method(:validate_localizable_number) do
+        return unless Spree::Config.enable_localized_number?
```

Comment:
Rather than early-returning shouldn't we skip all this module's behavior when number localization is not enabled?